### ISSUE TITLE
Verdaccio doesn't start because of wrong path

### ISSUE THREAD

**Issue by notsag** [verdaccio/ansible-verdaccio#28]
Verdaccio is installed in `/usr/bin/` but the systemd unit file try to use `/usr/local/bin/`

**Comment by juanpicado** [comment]
@030 it seems related https://github.com/verdaccio/ansible-verdaccio/pull/25

**Comment by 030** [comment]
@juanpicado @notsag Could you indicate what distro your are using and what version of this ansible role?

**Comment by notsag** [comment]
I used version 2.0.1 on ubuntu 18.04.

**Comment by 030** [comment]
@juanpicado I also think that it is related. @kai-chu could you have a look?

**Comment by 030** [comment]
```
● verdaccio.service - verdaccio Service
     Loaded: loaded (/etc/systemd/system/verdaccio.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sat 2020-12-19 02:50:35 CET; 2s ago
    Process: 46375 ExecStart=/usr/local/bin/verdaccio -c /home/verdaccio/verdaccio/verdaccio.yml (code=exited, status=203/EXEC)
   Main PID: 46375 (code=exited, status=203/EXEC)

dec 19 02:50:35 laptop22aug2020 systemd[1]: Started verdaccio Service.
dec 19 02:50:35 laptop22aug2020 systemd[46375]: verdaccio.service: Failed to execute command: No such file or directory
dec 19 02:50:35 laptop22aug2020 systemd[46375]: verdaccio.service: Failed at step EXEC spawning /usr/local/bin/verdaccio: No such file or directory
dec 19 02:50:35 laptop22aug2020 systemd[1]: verdaccio.service: Main process exited, code=exited, status=203/EXEC
dec 19 02:50:35 laptop22aug2020 systemd[1]: verdaccio.service: Failed with result 'exit-code'.
```

**Comment by 030** [comment]
@notsag Could you check whether the issue has been fixed in 2.1.1?

**Comment by notsag** [comment]
> @notsag Could you check whether the issue has been fixed in 2.1.1?

Yes it works like a charm now. I will remove the symlink I made as a workaround.
Thanks!

**Comment by juanpicado** [comment]
Thanks @030 you rock !!
